Consistent code formatting.

metonymic-smokey opened the pull request:
Run clang-format to enforce consistent formatting in the files changed.

abhinavdangeti reviewed (changes requested):
It's fine editing/formatting files we've added, but if we change those that we don't own - we're setting ourselves up for merge conflicts on any changes to them in the future.

metonymic-smokey commented:
> It's fine editing/formatting files we've added, but if we change those that we don't own - we're setting ourselves up for merge conflicts on any changes to them in the future.

I'm attempting to limit it to files we own entirely and functions we added within files we don't own.

abhinavdangeti commented:
Thanks @metonymic-smokey . Just the one comment about the funky indentation then and we can get this in.
